### DISCUSSION TITLE
Add DAB time to DLS log

### DISCUSSION THREAD

**Discussion by andimik** [KejPi/AbracaDABra#154] [Ideas]
The DLS log currently starts with 00:00:00, 

But although the file name of the `txt` file shows the start time it would be useful to have the DAB time/RTC logged as well. This is for example necessary if not all songs are displayed on DAB or when it's not transmitted at all on DAB but on the radio station's playlist on their website it's available).

# before

```
00:00:00	Lahko noč, otroci! - Igrajmo se gledališče   premiera pravljice Cvetke Sokolov
00:00:09	Vedno PRVI
00:01:24	OTROŠKI PEVSKI ZBOR OSNOVNE ŠOLE POLHOV GRADEC - LUNIN ROJSTNI DAN
00:03:11	MARC ANTONIE, PATTI AUSTIN - COULD YOU BE LOVED
```

# after

```
00:00:00	2024-09-12-195500	Lahko noč, otroci! - Igrajmo se gledališče   premiera pravljice Cvetke Sokolov
00:00:09	2024-09-12-195509	Vedno PRVI
00:01:24	2024-09-12-195624	OTROŠKI PEVSKI ZBOR OSNOVNE ŠOLE POLHOV GRADEC - LUNIN ROJSTNI DAN
00:03:11	2024-09-12-195811	MARC ANTONIE, PATTI AUSTIN - COULD YOU BE LOVED
```

**Comment by KejPi** [comment]
To be hoinest I do not understand your point.

**Comment by andimik** [comment]
Then we would have an absolute time as well. Currently we only have a relative time.

I have seen that on some local Slovenian radio stations, where DLS will not work for all songs (probably due to a communication loss), but on the internet (= the absolute time) you see all songs in the playlist.

**Comment by KejPi** [comment]
OK, I still do not understand why you need it but this feature can be implemented quite easily. I assume that the absolute time shall be DAB time, not local time on your computer, correct?

**Comment by KejPi** [comment]
Since I did not get any reply I have implemented this feature the way I think it should work.
